pages/about.md

```diff
@@ -35,21 +37,21 @@ permalink: /about/
     <section class="container">
 
       <div class="container-left-7">
-		  
+
 		  <h2>Who’s involved</h2>
-		  
-		  <h3>Multi-stakeholder group</h3>
-		  
+
+		  <h3 class="h4">1 | Multi-stakeholder group</h3>
```

Comment:
It needs to be h3 for semantics (the content should be in an outline from h1 on down), but I want the style to be like an h4. I think this is actually wrong on other pages, but I didn't want to go back through right now :grin: